poetry.lock

```diff
@@ -968,47 +968,47 @@ scipy = ["scipy"]
 
 [[package]]
 name = "numpy"
-version = "1.26.3"
+version = "1.26.4"
```

Comment:
Why `numpy` is upgraded ? Did you run `poetry lock --no-update` ?

Reply:
done

Reply:
I do not see any change.

Reply:
yes I did a `poetry lock --no-update` but when I add numpy it with a poetry command it upgrades it. I added it manually and now it should be good.